common/src/main/java/gov/cms/ab2d/common/repository/CoverageRepository.java

```diff
@@ -1,46 +1,29 @@
 package gov.cms.ab2d.common.repository;
 
-import gov.cms.ab2d.common.model.Beneficiary;
-import gov.cms.ab2d.common.model.Contract;
 import gov.cms.ab2d.common.model.Coverage;
+import gov.cms.ab2d.common.model.CoverageSearch;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Modifying;
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 
 import java.util.List;
-import java.util.Optional;
 
 @Repository
 public interface CoverageRepository extends JpaRepository<Coverage, Long> {
 
-
-    @Query(" SELECT c.beneficiary.patientId " +
+    @Query(" SELECT c.beneficiaryId " +
```

Comment:
With the use of Spring data, I don't see why this query needs to be specifically written.

Reply:
Missed that this is a scalar query, not an object one.